src/telcameras_v2/tools.py

```diff
@@ -38,3 +45,17 @@ def data_to_observation(data):
     observation['longitude'] = round(observation['longitude'], 13)
 
     return observation
+
+
+def store_data_for_sensor(data):
+    # Does the sensor exist and is it active
+    try:
+        sensor = Sensors.objects.filter(objectnummer=data.get('sensor', '')).get()
+    except Sensors.DoesNotExist:
+        logger.info(f"The sensor '{data['sensor']}' was not found, so the data is not stored.")
```

Comment:
We verbergen hier errors. Ze zijn wel terug te vinden in de logs, maar we krijgen er geen alert van. Is dat wenselijk?

Reply:
Is besproken. Dat is idd hoe we het willen hebben.